dev-python/matplotlib2tikz/matplotlib2tikz-0.6.6.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{4,5} )
```

Comment:
I currently don't have python3.6 installed on my system. I'll try to bump one of my system during the upcoming week. However, matplotlib2tikz is tested with 3.6 by upstream with travis.